src/Round.js

```diff
@@ -0,0 +1,25 @@
+// const Turn = require('../src/Turn');
+const Turn = require('./Turn');
+
+class Round {
+  constructor(deck) {
+    this.deck = deck;
+    this.currentCard = this.deck.cards[0];
+    this.turnCount = 0;
+    this.incorrectGuesses = [];
+    this.currentTurn;
+
+  }
+
+  returnCurrentCard() {
+    // console.log(this.deck)
+    return this.currentCard
+  }
+
+  takeTurn(guess) {
+    this.currentTurn = new Turn(guess, this.currentCard)
```

Comment:
For whatever reason, it gave me major headaches trying to figure out how to do this. But yay, it's done. Seems to work. I like it.